from0k2bp.tex

```diff
@@ -946,7 +964,7 @@
 proof]{\texorpdfstring{\protect\hypertarget{anchor-34}{}{}An inner
 product proof}{An inner product proof}}\label{an-inner-product-proof}}
 
-In Section 5 of Groth's paper, he presents the core algorithm, which
+In Section~5 of Groth's paper, he presents the core algorithm, which
```

Comment:
This is like the $N$~NUMS thing above; what is this?

Reply:
As previously commented. When PDF was rendered, `N` and `NUMS` was splited to two lines. Tilde prevents that.